tests/Benchmarks/HttpParserBench.cs

```diff
@@ -269,6 +369,54 @@ static int FullRequestLegacy()
     }
 }
 
+static class HttpParserExtensions
```

Comment:
move to separate file?

Reply:
I kind of prefer to keep all Http parsing code in this file. All other benchmarks are in one file each.